### PR TITLE
updating pw and fixing e2e full suite failures

### PR DESCRIPTION
**Pull request by mozrokafor**
# References: 
Jira: [MNTOR-2564](https://mozilla-hub.atlassian.net/browse/MNTOR-2564)

updating playwright version to the latest(1.40.0), also fixing some minor failures from the e2e full suite

[MNTOR-2564]: https://mozilla-hub.atlassian.net/browse/MNTOR-2564?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mansaj — APPROVED** [review]
LGTM pending nit

File: src/e2e/pages/landingPage.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class LandingPage {
     this.MozillaFooterLogoLink = page.locator("footer a").first();
     this.AllBreachesLink = page.getByRole("link", { name: "All Breaches" });
     this.FAQLink = page.locator('footer a:has-text("faq")');
-    this.TermsLink = page.getByRole("link", { name: "Terms & Privacy" });
+    this.TermsLink = page.getByRole("link", { name: "Terms of Service" });
```
Comment (pdehaan) [inline src/e2e/pages/landingPage.ts:46]: Unrelated to this PR, but now that we have separate "Terms of Service" and "Privacy Notice" footer links, do we have/need a test for also checking the Privacy Notice footer link? https://www.mozilla.org/privacy/subscription-services?

Also, probably not worth fixing, but there seems to be some inconsistency w/ naming between stuff like "ourMissionLink" and "TermsLink" where sometimes props have initial uppercase and sometimes initial lowercase (PascalCase vs camelCase).

Comment (mozrokafor) [reply src/e2e/pages/landingPage.ts:46]: good point here, will update

File: src/e2e/pages/landingPage.ts [hunk 2]
```diff
@@ -95,7 +96,7 @@ export class LandingPage {
   }
 
   async maybeClearBanner() {
-    if (await this.falseDoorBanner.isVisible()) {
+    if (await this.falseBannerCloseButton.isVisible()) {
```
[the remark this answers is not in the document]
Comment (mozrokafor) [reply src/e2e/pages/landingPage.ts:99]: great point!

**Pull request merged**